client/driver/fabfile.py

```diff
@@ -252,41 +266,43 @@ def save_dbms_result():
 def save_next_config(next_config, t=None):
     if not t:
         t = int(time.time())
-    with open(os.path.join(CONF['save_path'], '{}__next_config.json'.format(t)), 'w') as f:
+    with open(os.path.join(dconf.RESULT_DIR, '{}__next_config.json'.format(t)), 'w') as f:
         json.dump(next_config, f, indent=2)
     return t
 
 
 @task
 def free_cache():
-    cmd = 'sync; sudo bash -c "echo 1 > /proc/sys/vm/drop_caches"'
-    local(cmd)
+    if dconf.HOST_CONN != 'docker':
+        with show('everything'), settings(warn_only=True):  # pylint: disable=not-context-manager
+            res = sudo("sync && echo 3 | tee /proc/sys/vm/drop_caches")
```

Comment:
On my machine. This command causes permission deny.
I need to use `res = sudo("sh -c \"echo 3 > /proc/sys/vm/drop_caches\"")`